feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/PreviewScreenComponents.kt

```diff
@@ -65,24 +66,33 @@ import kotlinx.coroutines.CompletableDeferred
 private const val TAG = "PreviewScreen"
 
 /**
- * Displays a [Toast] with specifications set by a [ToastMessage].
+ * An invisible box that will display a [Toast] with specifications set by a [ToastMessage].
  *
  * @param toastMessage the specifications for the [Toast].
  * @param onToastShown called once the Toast has been displayed.
  */
 @Composable
 fun ShowToast(modifier: Modifier = Modifier, toastMessage: ToastMessage, onToastShown: () -> Unit) {
```

Comment:
nit: Maybe rename `ShowToast` to `ShowTestableToast` to make it more explicit why this composable needs to be called rather than just calling `Toast.show()` directly.